recipe/conda_build_config.yaml

```diff
@@ -8,7 +8,7 @@ MACOSX_SDK_VERSION:            # [osx and qt_version == 6]
 # As of 2024/03 -- qt6 works better than qt5 on aarch64. So drop qt5.
 # As of 2024/05 -- We had trouble gettin the aarch64 builds to work well with Qt6 and import cv2 on the CIs.
 #                  We can re-consider adding Qt6 support for Aarch upon request.
+# As of openCV 4.11 we dropped Qt5
 qt_version:
   - none
-  - 5          # [not (ppc64le or aarch64 or osx)]
```

Comment:
@traversaro are you OK with dropping Qt5 support? I would like to shed some burden off the build matrix

Reply:
Yes!

Reply:
(sorry I had missed the question)

Reply:
thanks for confirming